config.json

```diff
@@ -136,7 +136,14 @@
       "core": false,
       "difficulty": 4,
       "slug": "luhn",
-      "topics": null,
+      "topics": [
+      	"algorithms",
```

Comment:
Sorry to be very picky but this should have one more space of indentation I think. Would you mind fixing it? If you don't want to then that's fine, just let me know and I can fix it :)